### DISCUSSION TITLE
How do I hide the "Welcome to GitLens 13" page?

### DISCUSSION THREAD

**Discussion by riksagar** [gitkraken/vscode-gitlens#2372] [Q&A] [answered]
Is it a requirement to create a gitlens+ account and sign-in to use gitlens now?
I just installed gitlens on a new machine and there's nothing in the GitLens tab in VSCode other than a bunch of blurb about gitlens features.

When I click the "CLOSE" button at the bottom it closes the slide-out.

I can't find the familiar branches/commits/stashes/etc. info that I'm used to on  my other machines where I've been running GitLens in the past.

**Comment by d13** [comment, marked as answer]
> Is it a requirement to create a gitlens+ account and sign-in to use gitlens now?

GitLens is COMPLETELY FREE, NO ACCOUNT or anything, and can be used on ANY repo (local/public/private/etc).

**GitLens+ features on local & public repos** are also COMPLETELY FREE, NO ACCOUNT or anything. ONLY **GitLens+ features on private repos** require a Pro plan (or greater).

> I just installed gitlens on a new machine and there's nothing in the GitLens tab in VSCode other than a bunch of blurb about gitlens features.
>
> When I click the "CLOSE" button at the bottom it closes the slide-out.
>
> I can't find the familiar branches/commits/stashes/etc. info that I'm used to on my other machines where I've been running GitLens in the past.

By default, those GitLens views are located on the Source Control panel:
![Screen Shot 2022-11-23 at 1 31 18 PM](https://user-images.githubusercontent.com/293282/203622325-318bfcb1-96f8-4ebb-ba31-4af80403f3f3.png)

If you would like to move those views to the GitLens side panel, simply click the GitLens icon in the "Features" section of the Home view:
![Screen Shot 2022-11-23 at 1 33 09 PM](https://user-images.githubusercontent.com/293282/203622560-07650d4c-5d18-461e-85b5-af5d3ebf472a.png)

**Reply by ghost** [reply]
Thank you so much it worked.

**Comment by a-cuenca** [comment]
hey, have you figured it out? cause i have the same problem

**Reply by riksagar** [reply]
Hi.  Yes, the previous answer solves it.  You have to change the setting on the *Home* / *What's New* .

See below, I highlighted it. 

![Screenshot_20221204-083419_Firefox](https://user-images.githubusercontent.com/20214854/205503373-7b26ed03-c834-4527-a5d4-e06b5df4f8e0.jpg)

**Comment by riksagar** [comment]
Alternatively, if you leave it at the default setting, you'll find all the GitLens goodies up in the *Source Control Side Bar* (the thing below the Spy Glass).

**Comment by ruperson** [comment]
Still couldn't make this thing go away

**Reply by eamodio** [reply]
You can right-click on it and choose "Hide 'GitLens'"

**Comment by rkodirkhonov** [comment]
![image](https://user-images.githubusercontent.com/68010191/227584717-b6d60bcf-b69d-42c8-922e-b6e87f627456.png)

Just Tick off the Home as in the picture
